tests/leagueResource.spec.js

```diff
@@ -101,33 +101,33 @@ describe ("resource: leagueResource", function() {
   it ("should build a proper url to retrieve teams via a league key", function() {
     nock('https://fantasysports.yahooapis.com')
       .get("/fantasy/v2/league/328.l.34014/teams?format=json")
-      .reply(200, {});
+      .reply(200, require("./nock-data/leagueTeams"));
 
-    league.teams('328.l.34014', null);
+    league.teams('328.l.34014', () => {});
 
     expect(yf.api)
       .toHaveBeenCalledWith("GET", "https://fantasysports.yahooapis.com/fantasy/v2/league/328.l.34014/teams?format=json", jasmine.any(Function));
   });
 
-  // players
-  it ("should build a proper url to retrieve players via a league key", function() {
-    nock('https://fantasysports.yahooapis.com')
-      .get("/fantasy/v2/league/328.l.34014/players?format=json")
-      .reply(200, {});
+  // // players
+  // it ("should build a proper url to retrieve players via a league key", function() {
```

Comment:
This is [commented out](https://github.com/whatadewitt/yfsapi/blob/master/resources/leagueResource.mjs#L191) in the source, so there is nothing to test yet.